glossary.rmd

```diff
@@ -3,31 +3,31 @@ ottrpal::set_knitr_image_path()
 ```
 
 Computing Engine
-: A program that executes WDL on files. It may execute them your own laptop, a HPC cluster, or a Cloud Computing Platform
+: A program that executes WDL on files. It may execute them your own laptop, a HPC cluster, or a cloud computing platform. The two most commonly used computing engines are miniwdl and Cromwell.
 
 Container
-: A collection of software and dependencies that can run on nearly any machine
+: A collection of software and dependencies that can run on nearly any machine with the same hardware architecture.
```

Comment:
I added this caveat because images are loosely architecture specific. For example, I run Mac OS on Intel hardware, and when I build a Docker image I end up with linux-amd86 image. Here's one I pushed to Dockerhub.
<img width="1230" alt="Screenshot 2024-03-12 at 11 37 29 AM" src="https://github.com/fhdsl/WDL_Workflows_Guide/assets/27784612/b7655080-0643-4b27-98ad-b14d97cd3e1c">

If I were to ask someone else with Mac OS on Apple Silicon (which is ARM) to pull that image and run it, they'd run into issues, even though we're both using Mac OS and the image itself is Linux.

To solve this problem, many of the most popular images are multi-arch... 

<img width="1191" alt="Screenshot 2024-03-12 at 11 32 24 AM" src="https://github.com/fhdsl/WDL_Workflows_Guide/assets/27784612/d4d5d3d6-9307-486f-9c2d-783c44c2d7a9">


...but a lot of popular bioinformatics ones are just linux-amd64.
<img width="1210" alt="Screenshot 2024-03-12 at 11 35 31 AM" src="https://github.com/fhdsl/WDL_Workflows_Guide/assets/27784612/18bb13d3-cded-4994-ac9b-875e6520dfff">

I'm wondering if this concept is something we need to explain to users? I think these users would potentially need to know:
* Anyone using a 2020-or-later Macbook, which are all ARM
* Anyone using a Raspberry Pi or other linux-on-ARM scenario
* Anyone using an ARM Chromebook